ccloud/docs/ccloud-stack.rst

```diff
@@ -122,73 +122,61 @@ Create a ccloud-stack
-        User:119612      | ALLOW      | IDEMPOTENT_WRITE | CLUSTER          | kafka-cluster | LITERAL
-        User:119612      | ALLOW      | WRITE            | TRANSACTIONAL_ID | *             | LITERAL
-        User:119612      | ALLOW      | DESCRIBE         | TRANSACTIONAL_ID | *             | LITERAL
-        User:119612      | ALLOW      | WRITE            | GROUP            | *             | LITERAL
-        User:119612      | ALLOW      | CREATE           | GROUP            | *             | LITERAL
-        User:119612      | ALLOW      | READ             | GROUP            | *             | LITERAL
-        User:119612      | ALLOW      | DELETE           | TOPIC            | *             | LITERAL
-        User:119612      | ALLOW      | READ             | TOPIC            | *             | LITERAL
-        User:119612      | ALLOW      | WRITE            | TOPIC            | *             | LITERAL
-        User:119612      | ALLOW      | CREATE           | TOPIC            | *             | LITERAL
-        User:119612      | ALLOW      | DESCRIBE_CONFIGS | TOPIC            | *             | LITERAL
-        User:119612      | ALLOW      | DESCRIBE         | TOPIC            | *             | LITERAL
 
+        ServiceAccountId | Permission |    Operation     |     Resource     |     Name      |  Type
+      +------------------+------------+------------------+------------------+---------------+---------+
+        User:186607      | ALLOW      | DESCRIBE         | TRANSACTIONAL_ID | *             | LITERAL
+        User:186607      | ALLOW      | WRITE            | TRANSACTIONAL_ID | *             | LITERAL
+        User:186607      | ALLOW      | IDEMPOTENT_WRITE | CLUSTER          | kafka-cluster | LITERAL
+        User:186607      | ALLOW      | READ             | GROUP            | *             | LITERAL
+        User:186607      | ALLOW      | WRITE            | GROUP            | *             | LITERAL
+        User:186607      | ALLOW      | CREATE           | GROUP            | *             | LITERAL
+        User:186607      | ALLOW      | DESCRIBE         | TOPIC            | *             | LITERAL
+        User:186607      | ALLOW      | DELETE           | TOPIC            | *             | LITERAL
+        User:186607      | ALLOW      | CREATE           | TOPIC            | *             | LITERAL
+        User:186607      | ALLOW      | DESCRIBE_CONFIGS | TOPIC            | *             | LITERAL
+        User:186607      | ALLOW      | WRITE            | TOPIC            | *             | LITERAL
+        User:186607      | ALLOW      | READ             | TOPIC            | *             | LITERAL
 
    If you ran with ksqlDB (in the output below, ksqlDB service account ID is ``119613``):
 
    .. code-block:: text
-   
-        ServiceAccountId | Permission |    Operation     |     Resource     |             Name             |   Type    
+
```

Comment:
See line above, does it need to change (no ksqlDB SA)

>   If you ran with ksqlDB (in the output below, ksqlDB service account ID is ``119613``):

Reply:
Yes, thank you.  Will push a change with the other docs change